test/unity/toolkits/object_detection/test_object_detector.cxx

```diff
@@ -212,7 +212,7 @@ class test_object_detector: public object_detector {
 public:
   using create_iterator_call =
       std::function<std::unique_ptr<data_iterator>(
-          gl_sframe data, std::vector<std::string> class_labels, bool repeat)>;
+        data_iterator::parameters iterator_params)>; 
```

Comment:
Nit: could we avoid adding trailing whitespace, like the space after the semi-colon here? Might be useful to run git-clang-format